src/VastManager.js

```diff
@@ -426,7 +426,7 @@ var vastManager = function () {
 			};
 
 			var preroll = _player.currentTime() < 0.5;
-			if (preroll) {
+			if (preroll && _playlistIdx < 0) {
```

Comment:
I would rewrite this check to explicitly check `_playlistIdx === -1 ` –– Functionally it's identical but it's more conceptually clear that you are checking against a value with an explicit meaning, rather than a value in a range.

Reply:
Agreed with you. Code changed.

Reply:
👍